.github/workflows/ci.yaml

```diff
@@ -26,6 +26,14 @@ jobs:
       test: true
   build:
     uses: ./.github/workflows/build.yaml
+  publish-node-installer-image:
+    uses: ./.github/workflows/node-installer.yaml
+    needs: build
+    # This action requires use of the GITHUB_TOKEN to publish the image
+    # By default, PRs from forks don't have access, so we only run when the PR branch is on origin.
```

Comment:
Does this imply that maintainers of this project should push feat branches to origin?

Reply:
Yes.  Should I add a mention of this in other areas as well?  Maybe the README?

Reply:
I don't think it's necessary to put it in the README